.github/workflows/main.yaml

```diff
@@ -91,3 +96,42 @@ jobs:
         push: true
         tags: ${{ steps.docker_meta.outputs.tags }}
         labels: ${{ steps.docker_meta.outputs.labels }}
+  deploy:
+    runs-on: ubuntu-latest
+    needs: image_build
+    env:
+      DOCKER_TAG: ${{ needs.image_build.outputs.docker_tag }}
+    steps:
+    - uses: actions/checkout@v2
+    - uses: google-github-actions/setup-gcloud@master
+      with:
+        version: '290.0.1'
+        service_account_key: ${{ secrets.GKE_SA_KEY }}
+        project_id: ${{ secrets.GKE_PROJECT }}
+    - run: |-
+        gcloud container clusters get-credentials "$GKE_CLUSTER" --region "$GKE_REGION"
```

Comment:
can we remove the quotes around env variables, some things parse the quotes and others add them making it a potential liability if we change the runner